config/locales/en/lookup.yml

```diff
@@ -10,8 +10,10 @@ en:
         input_error: Enter a valid postcode
         error_description: Enter a valid postcode
     lookup:
-      input_label: Enter your postcode
-      hint_text: For example LS1 1UR
+      input_label: <h2 class="govuk-heading-s"> Enter a full postcode </h2>
```

Comment:
Will this put a `h2` tag inside a `label`? Only I'm pretty sure that's invalid HTML.

What I think you need is this: https://github.com/alphagov/govuk_publishing_components/pull/1747 which will go out in the next version of the components gem early next week, might be worth hanging on and waiting for that.

Reply:
> Has this css class been added because we want to make the label an "H2" but not actually change the visible size of it?

Hi Leena, yes it was. The default heading level for a h2 tag in the design system has the css "govuk-heading-l" whereas the card specified a small size, not ideal I agree.

Reply:
> Will this put a `h2` tag inside a `label`? Only I'm pretty sure that's invalid HTML.
> 
> What I think you need is this: [alphagov/govuk_publishing_components#1747](https://github.com/alphagov/govuk_publishing_components/pull/1747) which will go out in the next version of the components gem early next week, might be worth hanging on and waiting for that.

Hi Andy, that is great timing haha this is exactly what I needed! I will wait for the update 😄